contracts/core/Registry.sol

```diff
@@ -1,18 +1,35 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.0;
 
+import { OwnableRoles } from "../../lib/solady/src/auth/OwnableRoles.sol";
+import { Metadata } from "./libraries/Metadata.sol";
+
 // use Solady Roles for ownership of projects: https://github.com/Vectorized/solady/blob/main/src/auth/OwnableRoles.sol
-contract Registry {
+contract Registry is OwnableRoles {
+
+    // ASK: should this be renamed to identityIndex ?
+    uint private identityId;
```

Comment:
@zobront does it make sense to rename this to identityIndex

Reply:
Either one is fine. I prefer ID but your call.

Reply:
sticking with ID